<launch>
	<node pkg="tanwaylidar_view" name="TanwayLidar_MainNode" type="TanwayLidar_MainNode" output = "screen">
		<param name="frame_id" type="string" value="TanwayTP" />
		<param name="topic" type="string" value="/tanwaylidar_pointcloud" />
		<param name="imu_topic" type="string" value="/tanwaylidar_imu" />
		<param name="LidarType" type="int" value="21"/> 

		<!--连接模式:仅支持"on-line"、"off-line"两种模式配置-->
		<param name="ConnectType" type="string" value="on-line" />

		<!--实时模式下配置-->
		<!--本机IP地址-->
		<param name="LocalHost" type="string" value="192.168.111.204" />

		<!--回放模式下配置-->
		<!--PCAP文件绝对路径-->
		<param name="PcapFilePath" type="string" value="/home/tanway/lidar_data/TW360.pcap" />
		<param name="AlgoTablePath" type="string" value="$(find tanwaylidar_view)/sdk/config/algo_table.json" />
		
		<!--雷达设备的IP地址-->
		<param name="LidarHost" type="string" value="192.168.111.51" />
		<!--雷达发送点云数据的目的端口（即设备接收点云数据的端口）-->
		<param name="LocalPointloudPort" type="int" value="5600" />
		<!--雷达发送DIF数据的目的端口（即设备接收DIF数据的端口）-->	
		<param name="LocalDIFPort" type="int" value="5700" />
		<!--雷达发送IMU数据的目的端口（即设备接收IMU数据的端口）-->
		<param name="LocalIMUPort" type="int" value="5900" />


		<!--帧时间戳以第一个点云first_point、最后一个点云last_point-->
		<param name="FrameTimeStampType" type = "string" value = "first_point"/>
		<!--点云时间戳以雷达时间true、系统时间false-->
		<param name="UseLidarClock" type = "bool" value = "true"/>
		

	</node>
</launch>
